<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>薪酬管理系统</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>功能菜单</string>
     </property>
     <widget class="QPushButton" name="upload">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>导入考勤数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upload2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>导入绩效数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="importRoster">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>导入花名册</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addEmployee">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>添加员工</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backupDatabase">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>备份数据库</string>
      </property>
     </widget>
     <widget class="QPushButton" name="uploadReward">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>奖惩数据上传</string>
      </property>
     </widget>
     <widget class="QPushButton" name="userSettings">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>用户设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="socialInsurance">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>社保配置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="socialInsurance_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>薪酬项</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="backupPage">
      <widget class="QGroupBox" name="backupSettings">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>550</width>
         <height>300</height>
        </rect>
       </property>
       <property name="title">
        <string>备份设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>备份路径：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="backupPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="selectPath">
            <property name="text">
             <string>选择路径</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="enableAutoBackup">
          <property name="text">
           <string>启用自动备份</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>备份间隔（天）：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="backupInterval">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="backupNow">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>立即备份</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveSettings">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>保存设置</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="socialInsurancePage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>社保配置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="addInsuranceGroup">
              <property name="text">
               <string>添加配置组</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="insuranceTable">
            <property name="columnCount">
             <number>4</number>
            </property>
            <column>
             <property name="text">
              <string>配置组名称</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>社保所在地</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>缴纳基数</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>操作</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
